test/feature/win/parameters.yml

```diff
@@ -0,0 +1,61 @@
+# Receivers that agent needs to tests
+receivers: "host"
+
+#Test case name
+test_case: "win_feature"
+validate_type: "feature"
+# Only support metrics/traces/logs, even in this case we validate more than logs,
+# we only make this data_type as a placeholder
+data_type: "metrics"
+
+# Number of logs being written
+number_monitored_logs: 1
+# Number of metrics to be sent or number of log lines being written  each minute
+values_per_minute: "2"
+# Number of seconds the agent should run and collect the metrics. In this case, 1 minutes
+agent_collection_period: 60 
+
+cloudwatch_agent_config: "<cloudwatch_agent_config>"
```

Comment:
I dislike the config file based testing. I prefer code should contain validation. We should avoid having config files as much as possible. It is confusing for new people.

Reply:
I might be wrong about this or others can chime in:
* Code validation would not give a standard format and its depends on the owner's design so it increase more confusing IMO (e.g metric_value_benchmark, performance, etc) while configuration file is opposite. 
* Even though as you said, go is an easy language to understand, giving more abstraction is better though and it would increase the change files later on which would increase complexity. (e.g as you saw before, adding a flag in `metric_value_benchmark` would need to register every flag in other test 
* Duplicate code would be another thing to concern if we use code validation. 

Overall, the only thing that makes me thing me between use  `code validation` or `testing_dir` is the way we can run multiple configuration insides an EC2 Instances which we don't have much use case (e.g metric_collection_interval would be better to have a mix of unit test and preferable unit test more than integ test IMO)